File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(number int) (w string) {
	if number % 3 == 0 {
		w += "Pling"
	}
	if number % 5 == 0 {
		w += "Plang"
	}
	if number % 7 == 0 {
		w += "Plong"
	}
	if w == "" {
		w = strconv.Itoa(number)
	}
	return
}

File: cpp/raindrops/raindrops.cpp
#include "raindrops.h"
#include <string>

using namespace std;

namespace raindrops {
  string convert(int number) {
    string text = "";
    if (number % 3 == 0)
      text += "Pling";
    if (number % 5 == 0)
      text += "Plang";
    if (number % 7 == 0)
      text += "Plong";
    return text.empty() ? to_string(number) : text;
  }
}  // namespace raindrops
